### ISSUE TITLE
feature request: sort and stack items

### ISSUE THREAD

**Issue by Ninjistix** [mushroom8009/AutheryteHelper#4]
currently we can sort inventory, but items split don't stack. would be cool if we could sort and stack option

![image](https://user-images.githubusercontent.com/12572974/151120216-65c97879-e480-4000-8b5c-0f4b0c708aa2.png)

**Comment by mushroom8009** [comment]
thanks for the request
i'll see if minion supports it just in  inventory

**Comment by mushroom8009** [comment]
you have been waiting for a long time for this.
Finally, I was able to create the ability to stack the same item.
It will be Release soon.

**Comment by Ninjistix** [comment]
I have tested this and it works as of [1.6.6 ](2022022017) but I would also like to request an option to have just the sort and stack icon floating in it's own UI, would reduce the time required to open AH and the switch to [Jank] then to sort.

**Comment by mushroom8009** [comment]
It's not my cup of tea, so I'll make it a text command.

**Comment by Ninjistix** [comment]
yeah I guess if its a text command then I can easily make a macro to run said command when I want to sort inventory, I also tested it and it works on the saddle bag, and retainer inventory with no issues.
